main.rb

```diff
@@ -0,0 +1,151 @@
+require './classroom'
+require './person'
+require './rental'
+require './student'
+require './teacher'
+require './book'
+# rubocop:disable all
```

Comment:
- [ ] Linters are used to make the app professional and help you write clean code, so kindly avoid disabling the linters

Reply:
Thanks for your feedback